javascript/javascript1/week2/weather-wear.js

```diff
@@ -0,0 +1,30 @@
+// Using if...else
+
+function chooseTheClothes(temperature) {
+  if (temperature <= 0) {
+    return "You should wear a jacket, hat and gloves";
+  } else if (temperature > 0 && temperature <= 10) {
+    return "You should wear a coat";
+  } else if (temperature > 10 && temperature <= 20) {
+    return "You should wear a jacket";
+  } else {
+    return "You should wear a t-shirt";
+  }
+}
+const clothesToWear = chooseTheClothes(15);
+console.log(clothesToWear);
+
```

Comment:
This is very good that you have tried two ways, I admire the motivation. If you want to practice a bit more, I recommend to try switch () methode for this exercise.